contracts/deploy/l1/00_deploy_SequencerInbox.ts

```diff
@@ -0,0 +1,12 @@
+import { HardhatRuntimeEnvironment } from "hardhat/types";
```

Comment:
should we move `deploy` into `scripts`

nit: also never understood why these files need to start with a number, is it to enforce deployment ordering

Reply:
scripts should contains files that are invoked directly by users, but deploy is invoked by hardhat-deploy plugin;
yes the number is to enforce deployment ordering